.credo.exs

```diff
@@ -164,7 +164,7 @@
         {Credo.Check.Readability.ImplTrue, false},
         {Credo.Check.Readability.MultiAlias, false},
         {Credo.Check.Readability.SeparateAliasRequire, false},
-        {Credo.Check.Readability.SinglePipe, false},
+        {Credo.Check.Readability.SinglePipe, true},
```

Comment:
Hooray! There are a few other rules we can enable in follow up PRs to help.

Reply:
Definitely! This one just stuck out to me since i believe you've caught me breaking this style rule and I've flagged it a couple times on prs. It's easy to do when you `<3` the `|>` !